<Type Name="FaultCallback" FullName="System.Activities.FaultCallback">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="3b9968c74f3321146ac75a2163f5818646232cd7" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="zh-TW" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
  </Metadata>
  <TypeSignature Language="C#" Value="public delegate void FaultCallback(NativeActivityFaultContext faultContext, Exception propagatedException, ActivityInstance propagatedFrom);" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed FaultCallback extends System.MulticastDelegate" />
  <TypeSignature Language="DocId" Value="T:System.Activities.FaultCallback" />
  <TypeSignature Language="VB.NET" Value="Public Delegate Sub FaultCallback(faultContext As NativeActivityFaultContext, propagatedException As Exception, propagatedFrom As ActivityInstance)" />
  <TypeSignature Language="C++ CLI" Value="public delegate void FaultCallback(NativeActivityFaultContext ^ faultContext, Exception ^ propagatedException, ActivityInstance ^ propagatedFrom);" />
  <AssemblyInfo>
    <AssemblyName>System.Activities</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Delegate</BaseTypeName>
  </Base>
  <Parameters>
    <Parameter Name="faultContext" Type="System.Activities.NativeActivityFaultContext" />
    <Parameter Name="propagatedException" Type="System.Exception" />
    <Parameter Name="propagatedFrom" Type="System.Activities.ActivityInstance" />
  </Parameters>
  <ReturnValue>
    <ReturnType>System.Void</ReturnType>
  </ReturnValue>
  <Docs>
    <param name="faultContext">
      <span data-ttu-id="7607c-101">為檢查工作流程錯誤所提供的 <see cref="T:System.Activities.ActivityFaultContext" />。</span>
      <span class="sxs-lookup">
        <span data-stu-id="7607c-101">The <see cref="T:System.Activities.ActivityFaultContext" /> provided for inspecting the workflow fault.</span>
      </span>
    </param>
    <param name="propagatedException">
      <span data-ttu-id="7607c-102">導致這個錯誤的例外狀況。</span>
      <span class="sxs-lookup">
        <span data-stu-id="7607c-102">The exception that caused the fault.</span>
      </span>
    </param>
    <param name="propagatedFrom">
      <span data-ttu-id="7607c-103">擲回例外狀況並產生錯誤的活動。</span>
      <span class="sxs-lookup">
        <span data-stu-id="7607c-103">The activity that threw the exception and generated the fault.</span>
      </span>
    </param>
    <summary>
      <span data-ttu-id="7607c-104">表示當排程的活動或活動的其中一個子系進入 <see cref="F:System.Activities.ActivityInstanceState.Faulted" /> 狀態時所呼叫的方法。</span>
      <span class="sxs-lookup">
        <span data-stu-id="7607c-104">Represents the method that is called when a scheduled activity or one of its children enters the <see cref="F:System.Activities.ActivityInstanceState.Faulted" /> state.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
</Type>